src/components/TextOverlaySettingsBubble.js

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { MiradorMenuButton } from 'mirador/dist/es/src/components/MiradorMenuButton';
+import { Slider } from '@material-ui/core';
+import TextIcon from '@material-ui/icons/TextFields';
+import CloseIcon from '@material-ui/icons/Close';
+import SubjectIcon from '@material-ui/icons/Subject';
+import OpacityIcon from '@material-ui/icons/Opacity';
+import TextSelectIcon from './TextSelectIcon';
+
+/** Control text overlay settings  */
+const TextOverlaySettingsBubble = ({
+  windowTextOverlayOptions, imageToolsEnabled, textsAvailable, updateWindowTextOverlayOptions, t,
+}) => {
+  const {
+    enabled, visible, selectable, opacity,
+  } = windowTextOverlayOptions;
+  const [open, setOpen] = useState(true);
+  const [showOpacitySlider, setShowOpacitySlider] = useState(false);
+
+  if (!enabled || !textsAvailable) {
+    return null;
+  }
+  return (
+    <div style={{
+      backgroundColor: 'rgba(255, 255, 255, 0.8)',
+      borderRadius: 25,
+      position: 'absolute',
+      right: 8,
+      top: imageToolsEnabled ? 66 : 8,
+      zIndex: 999,
+    }}
+    >
+      {open
+      && (
+      <>
+        <div style={{
+          borderImageSlice: 1,
+          borderImageSource: 'linear-gradient(to bottom, rgba(0, 0, 0, 0) 20%, rgba(0, 0, 0, 0.2) 20% 80%, rgba(0, 0, 0, 0) 80% )',
+          borderRight: '1px solid rgba(0, 0, 0, 0.2)',
+          display: 'inline-block',
+          paddingRight: 8,
+        }}
+        >
```

Comment:
Nit: vielleicht in einer Zeile?

Reply:
Hm, das hat eslint so gewollt (`react/jsx-closing-bracket-location` Regel), sollen wir da die Presets verändern?

Reply:
Kannst du machen, wenn du magst, muss aber nicht unbedingt.